.github/workflows/regression.yml

```diff
@@ -10,8 +10,13 @@ on:
       - 'develop'
       - 'master'
 
+permissions:
+  contents: read
```

Comment:
Here you set the workflow permission to `read` and then for each job to `none`.
Why not set it to `none` on workflow level directly ?

Reply:
The contents read is required for cloning the repo. Now I notice the code isn't cloning the repo.

Reply:
So, should it be `none` at workflow level ?